app/design/frontend/base/default/template/algoliasearch/internals/configuration.phtml

```diff
@@ -35,7 +35,7 @@ $session = Mage::getSingleton('customer/session');
 $customerGroupId = $session->getCustomerGroupId();
 
 $priceKey = '.'.$currencyCode.'.default';
-if ($config->isCustomerGroupsEnabled($storeId) && $customerGroupId !== 0) {
```

Comment:
I believe this code was preventing transformHit in common.js from working correctly. 5505a50 changed the product and hit-item template to use group_0 rather than default, but because this file didn't change they were mis-matched